cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/StopProposalParameters.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.servlet.parameters;
+
+import java.io.UnsupportedEncodingException;
+import javax.servlet.http.HttpServletRequest;
+
+
+/**
+ * Parameters for {@link com.linkedin.kafka.cruisecontrol.servlet.EndPoint#STOP_PROPOSAL_EXECUTION}.
+ *
+ * <pre>
+ * Stop the proposal execution.
+ *    POST /kafkacruisecontrol/stop_proposal_execution?json=[true/false]&amp;review_id=[id]
+ * </pre>
+ */
+public class StopProposalParameters extends AbstractParameters {
+  private Integer _reviewId;
+  private boolean _twoStepVerificationEnabled;
+
+  public StopProposalParameters(HttpServletRequest request, boolean twoStepVerificationEnabled) {
+    super(request);
+    _twoStepVerificationEnabled = twoStepVerificationEnabled;
+  }
+
+  public StopProposalParameters(HttpServletRequest request, boolean twoStepVerificationEnabled, StopProposalParameters reviewedParams) {
```

Comment:
Where is `reviewedParams` used here?

Reply:
We should have passed it to the `super(request, reviewedParams)` -- good catch!